tests/onnxruntime/test_modeling.py

```diff
@@ -974,7 +974,7 @@ def test_stable_diffusion_model_on_rocm_ep_str(self):
     def test_load_model_from_hub_private(self):
         token = os.environ.get("HF_HUB_READ_TOKEN", None)
 
-        if token is None:
+        if not token:
```

Comment:
There's a `require_hf_token` decorator in https://github.com/huggingface/optimum/blob/main/optimum/utils/testing_utils.py#L88
it's used in some places but it checks for `HF_AUTH_TOKEN` instead of `HF_HUB_READ_TOKEN` 🤔 should we use that decorator instead ?

Reply:
Here the issue seems that the token is set as an empty string instead of being set to `None` (so the same issue will happen for `require_hf_token`). Let's see if this fixes it, in which case I'll update `require_hf_token`